api/v1beta1/openstackmachine_types.go

```diff
@@ -107,6 +114,10 @@ type OpenStackMachineStatus struct {
 	// +optional
 	Ready bool `json:"ready"`
 
+	// FloatingAddressFromPoolReady is true when the FloatingIP is ready.
+	// +optional
+	FloatingAddressFromPoolReady *bool `json:"floatingAddressFromPoolReady,omitempty"`
```

Comment:
I don't think we need this because we set a Condition. Unless I've missed something important, please could you remove it?